css/index.css

```diff
@@ -102,14 +128,45 @@ p {
 }
 
 .middle-content .boxes .box {
-    width: 12.5%;
-    height: 100px;
+    align-items: center;
     background: black;
-    margin: 20px 2.5%;
     color: white;
     display: flex;
-    align-items: center;
+    height: 100px;
     justify-content: center;
+    margin: 20px 2.5%;
+    width: 12.5%;
+}
+
```

Comment:
Good work using the parent to style the individual boxes and overriding the color.